package.json

```diff
@@ -26,12 +27,14 @@
     "@babel/preset-env": "^7.7.1",
     "@babel/preset-react": "^7.7.0",
     "@babel/runtime": "^7.7.2",
-    "@storybook/addon-a11y": "^4.1.18",
-    "@storybook/addon-actions": "^4.1.18",
-    "@storybook/addon-info": "^4.1.18",
-    "@storybook/addon-knobs": "^4.1.18",
-    "@storybook/addon-viewport": "^4.1.18",
-    "@storybook/react": "^4.1.18",
+    "@compositor/kit": "^1.0.47",
+    "@storybook/addon-a11y": "^5.3.0-beta.22",
+    "@storybook/addon-actions": "^5.3.0-beta.22",
+    "@storybook/addon-docs": "^5.3.0-beta.22",
+    "@storybook/addon-info": "^5.3.0-beta.22",
+    "@storybook/addon-knobs": "^5.3.0-beta.22",
+    "@storybook/addon-viewport": "^5.3.0-beta.22",
+    "@storybook/react": "^5.3.0-beta.22",
```

Comment:
Should we be using the beta version over the latest stable release?

Reply:
Once there is some consensus, I will correct this to use the latest stable. I was testing a feature of the 5.3.x family and forgot to revert this.

Reply:
These are reverted to `5.2.8`